fibnonacci-air/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+use std::borrow::Borrow;
+
+use p3_air::{Air, AirBuilder, AirBuilderWithPublicValues, BaseAir};
+use p3_field::PrimeField64;
+use p3_matrix::dense::RowMajorMatrix;
+use p3_matrix::MatrixRowSlices;
+
+const NUM_FIBONACCI_COLS: usize = 3;
+
+pub struct FibonacciRow<F> {
```

Comment:
They are not in the same row because you are using local, next. Should this be Cols instead?